apps/frontend/src/components/global/ExportSplunkModal.vue

```diff
@@ -45,7 +45,10 @@
                   Are you sure that you have configured it properly?
                 </span>
                 <br />
-                <span>The 'index' field requires a valid index name from the specified Splunk instance.</span>
+                <span
+                  >The 'index' field requires a valid index name from the
```

Comment:
please don't format it like this.  keep the braces with their tags.  put just the content into its own line.